Common/AR/Ar.md

```diff
@@ -0,0 +1,197 @@
+
+To create an application with the `ARKit` support:
+
+1. Downloads the sources of the [ArcGIS Runtime API Toolkit](https://github.com/Esri/arcgis-runtime-toolkit-qt).
+Sets the `AR_TOOLKIT_SOURCE_PATH` variable to the path of the sources of the toolkit.
+
+2. Installs the ArcGIS Runtime SDK for Qt version 100.6
+
+3. In Qt Creator, creates a new project "ArcGIS Runtime 100.6 Qt Quick C++ app" or "ArcGIS Runtime 100.6 Qt Quick QML app".
+
+4. In the projet file (.pro), includes the .pri file corresponding to the API used for the project:
+
+```
+include($$AR_TOOLKIT_SOURCE_PATH/Plugin/CppApi/ArCppApi.pri) // for C++ API
+```
+or
+```
+include($$AR_TOOLKIT_SOURCE_PATH/Plugin/QmlApi/ArQmlApi.pri) // for QML API
+```
+
+5. Updates the `Info.plist` file to request persmission for camera:
+
+```
+<key>NSCameraUsageDescription</key>
+<string>Camera access is needed for AR testing</string>
+```
+
+See [NSCameraUsageDescription Apple's documentation](https://developer.apple.com/documentation/bundleresources/information_property_list/nscamerausagedescription?language=objc) for details.
+
+6. The project is ready to be build and run in an iOS device. You need to verify the compatibility of your device
+with the ARKit.
+
+## ARCore installation
+
+For the details to uses `ARCore`, pleases refere to the [Google's documentation](https://developers.google.com/ar/).
+The details of the installation of the `ARCore` is describes in the
+[documentation](https://developers.google.com/ar/develop/c/enable-arcore).
+
+To create an application with the `ARCore` support:
+
+1. Downloads the sources of the [ArcGIS Runtime API Toolkit: https://github.com/Esri/arcgis-runtime-toolkit-qt.
+The `AR_TOOLKIT_SOURCE_PATH` variable refers to the path of the sources of the toolkit.
+
+2. Downloads and extracts the ARCore SDK for Android v1.10: https://github.com/google-ar/arcore-android-sdk/releases.
+This project contains the C header file (`arcore_c_api.h`) used to build the ARCore wrapper.
+The `AR_CORE_HEADER_PATH` variable refers to the path of the ARCore header files.
+
+3. Downloads the ARCore `aar` file from the Maven repository: https://maven.google.com/com/google/ar/core/1.10.0/core-1.10.0.aar
+This archive contains the native libraries used by the ARCore wrapper. Renames the extention file from `aar` to `zip` and
+extracts it.
+The `AR_CORE_BINARIES_PATH` variable refers to the path of the files of the ARCore native libraries.
+
+4. Installs the ArcGIS Runtime SDK for Qt version 100.6
+
+5. In Qt Creator, creates a new project "ArcGIS Runtime 100.6 Qt Quick C++ app" or "ArcGIS Runtime 100.6 Qt Quick QML app".
+
+6. In the projet file (.pro), includes the .pri file corresponding to the API used for the project:
+
+```
+include($$AR_TOOLKIT_SOURCE_PATH/ArCppApi.pri) // for C++ API
+```
+or
+```
+include($$AR_TOOLKIT_SOURCE_PATH/ArQmlApi.pri) // for QML API
+```
+
+7. Copy the gradle files from the ARCore SDK for Android to the `Android` dir in our project.
```

Comment:
Specifics would help clarify here. It isn't obvious which dir to copy to/from. For example:

> Copy the gradle files from `toolkit/Common/AR/3rdparty/arcore/gradle` to the `Android` dir in your project.

Reply:
Are there lines you need to add to your pro/pri files to reference the gradle/manifest files? Nothing mentioned here, but the examples have additions

Reply:
To be sure the explanations are correct, I implemented the "Redlands Fire Hydrants" samples in new projects from scratch and notes all the steps.